test/scrabble_test.rb

```diff
@@ -5,9 +5,46 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
-  def test_it_can_score_a_single_letter
-    skip
-    assert_equal 1, Scrabble.new.score("a")
-    assert_equal 4, Scrabble.new.score("f")
+  def setup 
+    @scrabble = Scrabble.new
+  end
+  
+  def test_it_can_score_nil_and_empty
+    assert_equal 0, @scrabble.score("")
+    assert_equal 0, @scrabble.score(nil)
+  end
+
+  def test_it_can_score_single_letters 
+    assert_equal 1, @scrabble.score("a")
+    assert_equal 4, @scrabble.score("f")
+  end
+
+  def test_it_can_score_word 
+    assert_equal 8, @scrabble.score('hello')
+  end
+
+  def test_seven_or_more_characters
+    assert_equal 44, @scrabble.score('jacuzzi')
+  end
+  
+  def test_seven_method 
```

Comment:
This test name could use a little more explication, like test_seven_method_returns_a_boolean_based_on_string_length... even though it gets a little long, that describes both the method & what the method is doing